Fixed a typo where cairo_ps_surface_set_size was called instead of
cairo_pdf_surface_set_size.
https://github.com/IoLanguage/io/pull/359
https://github.com/IoLanguage/io/commit/8cd5b5ad09a7707c859a7bdea342ae7ff2466e3c
--- addons/Cairo/source/_IoCairoPDFSurface.c.orig
+++ addons/Cairo/source/_IoCairoPDFSurface.c
@@ -64,7 +64,7 @@ IoObject *IoCairoPDFSurface_setSize(IoCairoPDFSurface *self, IoObject *locals, I
 	double w = IoMessage_locals_doubleArgAt_(m, locals, 0);
 	double h = IoMessage_locals_doubleArgAt_(m, locals, 1);
 
-	cairo_ps_surface_set_size(SURFACE(self), w, h);
+	cairo_pdf_surface_set_size(SURFACE(self), w, h);
 	CHECK_STATUS(self);
 	return self;
 }
